apps/SageAccountWeb/package.json

```diff
@@ -22,7 +22,7 @@
     "react-cookie": "4.0.0",
     "react-dom": "18.2.0",
     "react-easy-crop": "^5.0.7",
-    "react-router": "^7.1.1",
+    "react-router": "^7.5.3",
```

Comment:
Upgrade react-router everywhere -- they have recently made improvements to their lazy-loading functionality